modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -3095,18 +3095,7 @@
           {
             "name": "benefit_type",
             "in": "query",
-            "description": "Benefit Type for the appeal. Required if Decision Review is a Higher Level Review.",
-            "enum": [
```

Comment:
Why were these enums removed? Did they not end up showing in the documentation?

Reply:
nope. enum is not a thing for the swagger documentation.

Reply:
doesn't do anything

Reply:
in this context the benefit type is a parameter...not schema.

Reply:
Note: Drew and I discussed possibly adding the enum as schema and then referencing the schema in the parameters documentation.